src/components/InputTodo.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FaPlusCircle } from 'react-icons/fa';
+
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      invalid: false,
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
```

Comment:
- [ ] If the only field of `inputText` is `title`, then it may be better to just do something like 

```javascript
setInputText({
  title: e.target.value,
});
```
Reason: it's more explicit and easier for someone reading the code to know what's happening.

Reply:
Agreed, I'll be changing this in the next commit